content/tutorial/04-advanced-sveltekit/01-hooks/04-handleerror/README.md

```diff
@@ -13,32 +14,32 @@ export function handleError({ event, error }) {
 }
 ```
 
-If you navigate to `/the-bad-place`, you'll see this in action — the error page is shown, and if you open the terminal (using the button to the right of the URL bar), you'll see the message from `src/routes/the-bad-place/+page.server.js`.
+Nếu bạn chuyển đến `/the-bad-place`, bạn sẽ thấy điều này trong action - trang lỗi được hiển thị, và nếu bạn mở terminal (sử dụng nút ở bên phải của thanh URL), bạn sẽ thấy thông báo từ `src/routes/the-bad-place/+page.server.js`.
```

Comment:
"see in action" đây không liên quan gì tới SvelteKit action, nó là idiom thôi ấy bạn

```suggestion
Nếu bạn chuyển đến `/the-bad-place`, bạn sẽ thấy điều này diễn ra - trang lỗi được hiển thị, và nếu bạn mở terminal (sử dụng nút ở bên phải của thanh URL), bạn sẽ thấy thông báo từ `src/routes/the-bad-place/+page.server.js`.
```